prime-router/docs/schema_documentation/az-az-covid-19-hl7.md

```diff
@@ -131,7 +131,7 @@ RE|Remark
 **Reference URL**:
 [https://confluence.hl7.org/display/OO/Proposed+HHS+ELR+Submission+Guidance+using+HL7+v2+Messages#ProposedHHSELRSubmissionGuidanceusingHL7v2Messages-DeviceIdentification](https://confluence.hl7.org/display/OO/Proposed+HHS+ELR+Submission+Guidance+using+HL7+v2+Messages#ProposedHHSELRSubmissionGuidanceusingHL7v2Messages-DeviceIdentification) 
 
-**Table**: LIVD-SARS-CoV-2-2021-09-29
+**Table**: LIVD-SARS-CoV-2-2021-11-17
```

Comment:
I don't know the history, but there is a latest table file in the repo.  Are we not using that latest, so we do not have to change all the schemas?  I am not sure who you can ask about this.  If you find we are not using that latest file, maybe delete it?

Reply:
Looks like Mo had created that file for the LIVD table endpoint (#2646).  I agree we have somewhat of a disconnect here with what's referenced different places.  Seems that we should either point all the schemas to the 'latest' file or have the MetaDataFunction reference the datestamped files like the schemas do.  I'd recommend the former.  Cleaner approach.

Reply:
Yes, I like the idea of having the schemas point at the latest. It's killing two birds with one stone.